### DISCUSSION TITLE
Mac使用PicGo中安装插件不成功！

### DISCUSSION THREAD

**Discussion by sunsjay347** [Molunerfinn/PicGo#601] [Q&A] [answered]
系统:Mac Big Sur 
PicGo版本：最新的稳定版本和体验版本都尝试了一下
node.js也安装了
问题：首先在插件中心搜索国内不搭梯子好像搜索不到。搭建梯子后可以搜索到，点击安装后就一直卡在安装中。

**Comment by Molunerfinn** [comment, marked as answer]
1. 插件中心的插件列表来自npm官方的搜索api。国内可能会比较慢，这个无解。
2. 卡在安装中也是类似的原因，国内npm安装在某些情况下确实比较慢。

后续会提供添加npm源的方案，可以通过国内源来下载，这样只要知道插件名字就能安装
